vms/rpcchainvm/vm_server.go

```diff
@@ -391,11 +389,7 @@ func (vm *VMServer) CreateStaticHandlers(ctx context.Context, _ *emptypb.Empty)
 }
 
 func (vm *VMServer) Connected(ctx context.Context, req *vmpb.ConnectedRequest) (*emptypb.Empty, error) {
-	nodeID, err := ids.ToNodeID(req.NodeId)
-	if err != nil {
-		return nil, err
-	}
-
+	nodeID := ids.NodeIDFromBytes(req.NodeId, len(req.NodeId))
```

Comment:
This should error if the bytes aren't 32 or 20

Reply:
Done